test/basic/vpc_create/vpc_create_test.go

```diff
@@ -13,8 +14,18 @@ import (
 
 	cac "github.com/aptible/cloud-api-clients/clients/go"
 	"github.com/aptible/terraform-provider-aptible-iaas/internal/client"
+	"github.com/aptible/terraform-provider-aptible-iaas/test/utils"
 )
 
+func cleanupAndAssert(c context.Context, t *testing.T, terraformOptions *terraform.Options, environmentId, assetId string) {
+	terraform.Destroy(t, terraformOptions)
+
+	resources, err := utils.GetTaggedResources(c, environmentId, assetId)
```

Comment:
i think you want to use this in a truthy case too, to make sure that it actually captures resources in a length that you expect in the TestVPCCreate, so you'd have something like:

```
assert.Equal(t, 10, len(resources)) // or maybe just a value greater than 0?
```

in that stanza, you'd never know if this doesn't work if you didnt do this